usr/share/lib/ipa/tests/SLES/infra/test_sles_force_new_smt_reg.py

```diff
@@ -0,0 +1,35 @@
+
+
+def test_sles_force_new_smt_reg(CheckCloudRegister,
+                                GetReleaseValue,
+                                GetSMTServers,
+                                host,
+                                request):
+    certs_dir = '/var/lib/regionService/certs/'
+    provider = request.config.getoption('provider')
+    region = request.config.getoption('region')
+
```

Comment:
This makes the assumption that regionserver client is of at least version 7.0.6 which is OK as we are looking forward from an image testing perspective. However it is probably reasonable to add a rudimentary version check on the rpm and if version is < 7.0.6 skip this test rather than have it fail and let the user fish for the reasons of the failure.